src/azul/transformer.py

```diff
@@ -300,6 +300,23 @@ def get(self):
         return thaw(super().get())
 
 
+class FrequencySetAccumulator(SetAccumulator):
```

Comment:
Don't like extending SetAccumulator because we'd be instantiating a `set` only to overwrite it with a `Counter`. Also see how max_size should be mandatory using a FrequencySetAccumulator without a max. size makes little sense. One should use a `SetAccumulator` instead.

Also, a one-liner docstring might not hurt.